testnet/stacks-node/src/config.rs

```diff
@@ -362,6 +362,9 @@ impl Config {
                     microblock_frequency: node
                         .microblock_frequency
                         .unwrap_or(default_node_config.microblock_frequency),
+                    microblock_attempt_time_ms: node
+                        .microblock_attempt_time_ms
+                        .unwrap_or(default_node_config.microblock_attempt_time_ms),
```

Comment:
What does `microblock_attempt_time_ms` control? Can you add something to the Mining section in `README.md` that describes the new config option?

Reply:
It controls how long the miner will spend iterating through transactions when building a microblock (soft deadline).  It's analogous to `first_attempt_time_ms` and `subsequent_attempt_time_ms` in the `[miner]` section.

Reply:
Also, it should be in `config.miner`.  Will move.